src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -4,5 +4,6 @@
 
 Rule ID | Category | Severity | Notes
 --------|----------|----------|-------
+CA1849 | Performance | Disabled | UseAsyncMethodInAsyncContext, [Documentation]
```

Comment:
Was this file manually edited?? The codefix should have produced a link.

Reply:
Ah, it is just bad merging, good catch!